### PR TITLE
Refactor code

### PR DESCRIPTION
**Pull request by UtopiaBeam**
- globally export `UserModule`
- add `SuspendOrActivateUserDTO`
- add `required: true` for required fields

### CODE REVIEW

File: src/admin/admin.controller.ts [hunk 1]
```diff
@@ -1,19 +1,20 @@
...
-        return this.service.suspend(id);
+    suspend(@Body() { userId }: SuspendOrActivateDTO) {
+        return this.service.suspend(userId);
     }
 
     @Patch('activate')
-    activate(@Body('id') id: string) {
-        return this.service.activate(id);
+    activate(@Body() { userId }: SuspendOrActivateDTO) {
+        return this.service.activate(userId);
```
Comment (beesk135) [inline src/admin/admin.controller.ts]: what does DTO help here? Is it about and option in API Prop? And if it is, what will happen after request Body not matching with DTO?

File: src/auth/auth.module.ts [hunk 2]
```diff
@@ -10,7 +9,6 @@ import { AuthController } from './auth.controller';
         JwtModule.registerAsync({
             useClass: ConfigService,
         }),
-        UserModule,
```
Comment (beesk135) [inline src/auth/auth.module.ts]: Why did u remove this UserModule which the exact same thing you removed from AdminSerivce

Comment (beesk135) [reply src/auth/auth.module.ts]: Sorry I have just seen `@Global` in user module file.

File: src/admin/admin.controller.ts (hunk 1 above)
Comment (UtopiaBeam) [reply src/admin/admin.controller.ts]: Right now it has no effect, but making it as a class is helpful when validate data.

**Pull request merged**